pinot-common/src/main/java/org/apache/pinot/sql/parsers/rewriter/QueryRewriterFactory.java

```diff
@@ -33,7 +33,7 @@ private QueryRewriterFactory() {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(QueryRewriterFactory.class);
 
-  static final List<String> DEFAULT_QUERY_REWRITERS_CLASS_NAMES =
+  public static final List<String> DEFAULT_QUERY_REWRITERS_CLASS_NAMES =
```

Comment:
Oh this is made visible also for us to conveniently configure in our config files. Intentionally left ArgMinMaxRewriter out because we don't want to add name check overhead to default codepath.